drink/Cargo.toml

```diff
@@ -21,5 +21,8 @@ parity-scale-codec = { workspace = true }
 scale-info = { workspace = true }
 thiserror = { workspace = true }
 
+[dev-dependencies]
+wat = { workspace = true }
```

Comment:
Wat is das?

Reply:
can turn wat to a wasm blob :man_shrugging:

Reply:
Why did you have to write a wat by hand?

Reply:
1. pallet-contracts' unit tests also use wat instead of full Rust contract compiled to wasm
2. no need for compiling contract to wasm / no need to keep obscure bytes
3. kinda readable
4. you don't have to encode any selectors or arguments: `data: vec![]` works well for calling the only method